### DISCUSSION TITLE
Relay

### DISCUSSION THREAD

**Discussion by samuelcastro** [UnlyEd/next-right-now#199] [Ideas] [answered]
Have you ever considered using [Relay](https://relay.dev/)?

https://github.com/vercel/next.js/tree/canary/examples/with-react-relay-network-modern

What's your thoughts on it?

I personally never worked with it but I always wondered how it would be.

**Comment by Vadorequest** [comment, marked as answer]
I have no experience whatsoever with it, I don't know the differences between Relay and ApolloClient.

Since I don't actively use GraphQL anymore, it's not likely I'd change the current implementation, even if it was better than ApolloClient. (but the community might)
